Gems/Atom/Tools/AtomToolsFramework/Code/Include/AtomToolsFramework/Viewport/ModularViewportCameraController.h

```diff
@@ -150,9 +150,10 @@ namespace AtomToolsFramework
         struct CameraAnimation
         {
             //! The transform of the camera at the start of the animation.
-            AZ::Transform m_transformStart = AZ::Transform::CreateIdentity();
-            AZ::Transform m_transformEnd = AZ::Transform::CreateIdentity(); //!< The transform of the camera at the end of the animation.
+            AZ::Transform m_transformStart;
+            AZ::Transform m_transformEnd; //!< The transform of the camera at the end of the animation.
             float m_time = 0.0f; //!< The interpolation amount between the start and end transforms (in the range 0.0 - 1.0).
+            float m_duration = 1.0f; //!< The length of the animation (how long it takes to completed).
```

Comment:
Nit: Would be nice to either name the variable or list in the comments what unit the duration is in. Milliseconds? Seconds? Ticks?